### ISSUE TITLE
A question for the parameters during the trainig

### ISSUE THREAD

**Issue by OwXiaoM** [Coloquinte/torchSR#4]
Hi, I am really grateful that you give us a very clear implementation of these SR models, by the way I would like to ask you about the parameters(optimiser, data-augement, epochs ,etc) that you have choosen in the reproduction during the training, especially for the EDSR huge, because I have problems when I apply the parameters in the paper on your training models, as I could not reach the pnsr and ssim as high as the paper. Your response can be really helpful for me and I would like to thank you in advance.

**Comment by Coloquinte** [comment]
Hi,
Thank you for your kind feedback -- and sorry for the delay.

I didn't reproduce the huge EDSR training, or at least I got rid of the results, so there's no guarantee there is no discrepancy.
I think the data augmentation, learning rates, batch size, etc. are all the same by default, so normally you should get similar results at the end. However the validation step during training is not the same, so expect different reports.

To test your network once trained, please use the --shave-border <SCALE> --eval-luminance options, as shown in the eval.sh scripts: this makes the evaluation using the same PSNR measure as NTIRE/EDSR.

**Comment by Coloquinte** [comment]
I think the command should just be:
```
python -m torchsr.train --arch $arch --scale $scale --tune-backend \
        --log-dir logs_train/${arch}_x${scale} --save-checkpoint ${arch}_x${scale}.pt
```
with everything else left at default.

Then for eval:
```
python -m torchsr.train --validation-only --arch $arch --scale $scale --dataset-val $dataset --chop-size 400
--load-pretrained ${arch}_x${scale}_model.pt --shave-border $scale --eval-luminance
```

**Comment by OwXiaoM** [comment]
Thank you enormously, I will contact you later if I find any interesting results during the usage of your code, and actually your program helps me really a lot, I believe oneday a Sota model will be born during the usage of your code, I will also try hard to make it too.

**Comment by OwXiaoM** [comment]
And I stil find something difference in your code comparing with the papers' experiments, perhaps theses changes make the results a little different, in EDSR, the author says he halves the learining rate every 2*10^5 minibatch, and in your code the default setting for the decay ratio is 0.1 for evrey 2*10^5 minibatch. And in your validation mode, the input patches is 300*300 by defaults rather than the full patches. I think perhaps because of these difference that the final result is not identical. But I have to say that your code is wonderful, which is really a very very meaningful project.

**Comment by Coloquinte** [comment]
Indeed, my learning rate adjustment was wrong. Add --lr-decay-rate 2 to the command.

The input patch size is correct for the standalone validation ([line](https://github.com/Coloquinte/torchSR/blob/d110fa03171d4421c9eb1ac866ab6137533dac9b/torchsr/train/helpers.py#L74))

If you want something more state-of-the-art, NinaSR is better, although I don't have anything on this repo for photo-realistic rendering.
